### ISSUE TITLE
Create Donate form 

### ISSUE THREAD

**Issue by Sanju-github002** [SahityaRoy/E-Learning-freesite#38]
Can I create Donate form by help google form or html and css which you like on your website ???

**Comment by vchsi** [comment]
I can create a Donate form with a Flask/Django backend and a HTML and CSS frontend. May I work on this issue?

**Comment by SahityaRoy** [comment]
Yes you can but I want to say that create a registration form using
back-end

**Comment by vchsi** [comment]
Do you want the form responses to go in a DB, or did you mean something else?

**Comment by SahityaRoy** [comment]
yes its store data in a database

**Comment by vchsi** [comment]
Ok, I can do this issue. Give me DB Type (mysql, pgsql, sqlite) and then the DB Access URL.

**Comment by SahityaRoy** [comment]
opps their is a problem like its a static website so this site does not
support DB

**Comment by vchsi** [comment]
If the website doesn't support databases, How will data get stored for registration/donation forms?

**Comment by SahityaRoy** [comment]
Donation Form with payment gate a way

**Comment by asitsonawane** [comment]
I can do this, please assign me.

**Comment by Roshani9731** [comment]
@SahityaRoy  Can I work on this?

**Comment by neeraj542** [comment]
Can I work on this issue

**Comment by shrutimeg** [comment]
@SahityaRoy I want to work on this and solve this issue
